lib/server.js

```diff
@@ -34,15 +34,21 @@ module.exports = class HFServer {
       wsURL,
     })
 
+    const defaultOrders =  [AccumulateDistribute, TWAP, Iceberg, PingPong]
```

Comment:
Can you move this to the top of the file as a constant `DEFAULT_ORDERS`?

Reply:
done!